opus/application/static_media/js/opus.js

```diff
@@ -767,6 +767,7 @@ var opus = {
         let adjustTableSizeDB = _.debounce(o_browse.adjustTableSize, 200);
         let adjustProductInfoHeightDB = _.debounce(o_cart.adjustProductInfoHeight, 200);
         let adjustDetailHeightDB = _.debounce(o_detail.adjustDetailHeight, 200);
+        let adjustDetailImgNavBtnsDB = function() {o_detail.adjustDetailImgNavBtns();};
```

Comment:
Isn't this just:
let adjustDetailImgNavBtnsDB = o_detail.adjustDetailImgNavBtns();